### PR TITLE
Allow a guest to become a host

### LINKED ISSUE #49: Allow a guest to become a host
# REQUIREMENTS
A guest must have an option to become a host by adding the necessary information

# Acceptance Criteria
List all required acceptance criteria
- [ The link "hazte anfitrión" should redirect to "users edit"] Criteria 1
- [ Users edit should have an option to upload the DNI in "users edit"] Criteria 2

### PR DESCRIPTION
**Pull request by Frcomparan**
# Description
What did you implemented/Why did you implemented this:
What?
- Made the "hazte anfitrión" link functional
- Changed permissions for unverified users (can't create homes)
- Added an option to become a host in edit view

Why?
- To allow guests to become hosts if they so choose
- To prevent unverified users from creating homes
  
  ## Screenshots 
Captura de pantalla 2022-05-24 193217

### CODE REVIEW

File: app/assets/stylesheets/views/devise/_edit_user.scss [hunk 1]
```diff
@@ -58,6 +62,11 @@
     }
   }
 
+  #dni {
+    width: 100%;
+    height: 100%;
```
Comment (JuanequeX) [inline app/assets/stylesheets/views/devise/_edit_user.scss]: sort this block alphabetically

File: app/views/devise/registrations/edit.html.erb [hunk 2]
```diff
@@ -49,6 +49,11 @@
         <%= f.select :gender, [["Masculino", "male"], ["Femenino", "female"]] %>
       </div>
 
+      <div class="field">
+          <%= f.label :role, 'Usuario' %>
+          <%= f.select :role, [["Huesped", "guest"], ["Anfitrión", "host"]], {}, onchange: "showDni();" %>
+        </div>
```
Comment (JuanequeX) [inline app/views/devise/registrations/edit.html.erb]: the identation could be 2 spaces, please fix it!

**Review by JuanequeX — DISMISSED** [review]

File: app/assets/stylesheets/views/devise/_edit_user.scss (hunk 1 above)
Comment (Frcomparan) [reply app/assets/stylesheets/views/devise/_edit_user.scss]: Sorted!

File: app/views/devise/registrations/edit.html.erb (hunk 2 above)
Comment (Frcomparan) [reply app/views/devise/registrations/edit.html.erb]: Fixed!

**Review by YilKanda — DISMISSED** [review]
LGTM!

**Review by brocha05 — DISMISSED** [review]

**Review by brocha05 — APPROVED** [review]

**Review by DiegoSteve — APPROVED** [review]
LGTM!

**Pull request merged**